addons/misc/functions/fnc_FAK_unpack.sqf

```diff
@@ -23,12 +23,31 @@ params ["_unit", "_item", "_type", "_slot"];
 private _fnc_arrayToInvItem = {
     params ["_unit", "_array", "_container"];
 
+    private _groundContainer = nearestObjects [_unit, ["WeaponHolder_Single_F"], 2] param [0];
```

Comment:
This should be replaced with a search for all ground containers as a created container will block access to another container if they are in close proximity. So a check for all the following containers:
```"GroundWeaponHolder",
"GroundWeaponHolder_Scripted",
"Library_WeaponHolder",
"WeaponHolder",
"WeaponHolder_Single_F",
"WeaponHolder_Single_limited_item_F",
"WeaponHolder_Single_limited_magazine_F",
"WeaponHolder_Single_limited_weapon_F",
"WeaponHolderSimulated",
"WeaponHolderSimulated_Scripted"
```
should be considered. This however could impact performance very slightly.

Reply:
Implemented in commit https://github.com/KAT-Advanced-Medical/KAM/pull/456/commits/d5b3975a6b86a561f73c44e2f5ad8c3f1d8f938f.